src/compositor/hwcinterface.h

```diff
@@ -0,0 +1,156 @@
+// IMPORTANT!!!!
+//
+// This file is a copy from the one in the hwcomposer platform plugin
+// Make sure it is up to date, failing to do so will result in the
+// hardware compositor not working or at worst, binary incompatibility.
```

Comment:
uint32 version; field at the start of the structs, maybe? (and an qFatal if they mismatch)